external/emulation/emulation/zhao_carr.py

```diff
@@ -90,10 +95,36 @@ def mask_where_fortran_cloud_identical(state, emulator):
         state[Input.cloud_water],
         emulator[GscondOutput.cloud_water],
     )
-    net_condensation = cloud_out - state[Input.cloud_water]
-    return {**emulator, **apply_condensation_liquid_phase(state, net_condensation)}
+    return _update_with_net_condensation(cloud_out, state, emulator)
+
+
+def _get_classify_output(emulator, class_name):
+    names = sorted(CLASS_NAMES)
+    idx = names.index(class_name)
+    logit_classes = emulator["gscond_classes"]
+    # TODO this should probably be in transforms
+    one_hot = logit_classes == np.max(logit_classes, axis=0, keepdims=True)
```

Comment:
It's a bit confusing that this is transposed from the fv3fit conventions where the one hot encoded dimension is the last one. Do you think it would be clearer to pass axis as a function argument (e.g. one_hot_axis=0)?

Reply:
Agree, thanks for the general discussion about the fv3fit/prognostic run ordering conventions.  I added the argument for now to add a bit of clarity.